Dockerfile

```diff
@@ -82,6 +87,14 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
+# Install Criu
+ENV CRIU_VERSION 1.7
```

Comment:
Yup, that's the story. It was really difficult to track down the problem with 1.8 because we weren't getting any failures outside of CI.